### ISSUE TITLE
Cannot get 27.0 map@tiou=0.5 after train without changing any code.

### ISSUE THREAD

**Issue by JacobKong** [Pilhyeon/BaSNet-pytorch#6]
After train the BaSNet, I only get 26.3 map@tiou=0.5 after train without changing any code. Could you give me any hit to reproduce the result?

Maybe the seed? I don't know.

The following are eval curve and the final test result.
![image](https://user-images.githubusercontent.com/10401849/70504406-b07cca00-1b60-11ea-8e76-9c07a3ccc261.png)

![image](https://user-images.githubusercontent.com/10401849/70505139-38170880-1b62-11ea-98eb-25dabadb18d6.png)

**Comment by Pilhyeon** [comment]
Performances can be a little different due to the statistical randomness.
So, if you want to reproduce the performance of the paper, we recommend you to use the pre-trained model we provided.
Thank you.

**Comment by JacobKong** [comment]
Thanks a lot!
